app/views/transactions/index.html.erb

```diff
@@ -0,0 +1,16 @@
+<%= render 'common/navigation', :title => 'Transactions' %>
```

Comment:
Please add a "Back" button (<) on `Transactions` page so when the user clicks it, it navigates the user to the home page.